cluster-install/install.sh

```diff
@@ -1621,28 +1678,43 @@ step_check_all_images_can_be_pulled() {
 step_check_astra_control_reachable() {
     logheader $__INFO "Checking if Astra Control is reachable at '$ASTRA_CONTROL_URL'..."
     make_astra_control_request "/"
-    if [ "$_return_status" == 200 ]; then
+    local -r status="$_return_status"
+    local -r body="$_return_body"
+    local err="$_return_error"
+    if [ "$status" == 200 ]; then
         logdebug "astra control: OK"
     else
-        add_problem "astra control: failed ($_return_status)" "Failed to contact Astra Control (status code: $_return_status)"
+        if [ "$status" != 000 ] || [ -z "$err" ]; then err="$(status_code_msg "$status")"; fi
+        logdebug "body: '$body'"
+        add_problem "Failed to contact Astra Control: $err ($status)"
```

Comment:
This and the few other changes below make use of the new `__ERR_FILE` stderr thing to improve error handling/logging for the Astra Control requests.